src/Microsoft.PowerShell.Commands.Utility/commands/utility/CSVCommands.cs

```diff
@@ -1278,9 +1279,12 @@ internal void ReadHeader()
         {
             _alreadyWarnedUnspecifiedName = alreadyWriteOutWarning;
             ReadHeader();
+            var prevalidated = false;
+            var values = new List<string>(16);
+            var builder = new StringBuilder(256);
```

Comment:
@powercode Please explain why we use this init constants? (Why not 32 and 512?) I believe we need to add a good comment.

Also above we use the same values. I believe we should use `readonly const`.

Reply:
They are just reasonable guestimates of number of columns and line length - just so that we don't have to resize and reallocate so much.

Agree that we can make constants of them.

Reply:
Fixing.

Reply:
I meant that the constants is used in ReadHeader() and Import().
After we read header we already know needed size of the `values` (header column count) and could use it in Import().
ReadHeader() is called once - seems make sense use init value only if we get a header from Header parameter - in the case we know size of the header.